lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorException.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Not sure what the convention is, but it seems like most other files use block comments.